importer/api/__init__.py

```diff
@@ -148,7 +148,7 @@ def import_children(course, element, parent, parent_dpath):
         None
     """
     # pylint: disable=too-many-locals
-    title = element.attrib.get("display_name", "MISSING")
+    title = element.attrib.get("display_name", "...")
```

Comment:
I was using unicode, But I revert to "..." as our discussion on hangout. I will open a issue for haystack is not accepting unicode character.